test/unit/fixtures/handlebars_features_test.adoc.hbs

```diff
@@ -0,0 +1,395 @@
+== {{ page.name }}
```

Comment:
Do you really want to just put this in one giant file? Or would it be better with each test in its own file? If its easy to diagnose failures with the one big file, then fine

Reply:
I would have multiple files and unit tests if this was somewhat more independent from mrdox. But for now I just have a few macros I'm using, no test framework, and it doesn't scale very well.

Reply:
Yeah that's fair. We will need to build up the tests (for mrdox and handlebars).